deps/v8/tools/js2c.py

```diff
@@ -32,7 +32,7 @@
 # library.
 
 import os, re, sys, string
-import optparse
+import argparse
```

Comment:
The Centos CI boxes are failing because of this. I suspect the Centos dists we are using use Python 2.6, while `argparse` was added in Python 2.7.